__tests__/app.test.js

```diff
@@ -234,3 +234,82 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+// PATCH
+describe("PATCH /api/articles/:article_id", () => {
+  test("status: 200 - responds with patched article with positive votes", () => {
+    const newArticleVotes = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newArticleVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(3);
+        expect(article.votes).toBe(10);
+      });
+  });
+  test("status: 200 - responds with patched article with negative votes", () => {
+    const newArticleVotes = {
+      inc_votes: -10,
+    };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newArticleVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(3);
+        expect(article.votes).toBe(-10);
+      });
+  });
+  test("status: 404 - responds with 404 if article does not exist", () => {
+    const newArticleVotes = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch("/api/articles/999")
+      .send(newArticleVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("not found");
+      });
+  });
+  test("status: 400 - responds with error for incorrect body shape", () => {
+    const newArticleVotes = {};
```

Comment:
This test is fine as 400 here, but could be a 200 returning the unchanged article, so in the future other properties could be patched as well without the need to include inc_votes with the existing value in the patch object every time.